mantidproject/mantid
# Use sphinx linking mechanism

This is a minor cleanup of the index pages to use the built-in linking mechanism of sphinx rather than hard-coded urls.

**To test:**

Build the documentation and see if the links still work.

*There is no associated issue.*

*Does not need to be in the release notes* because it is just a cleanup of links.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
